### PR TITLE
SPUI-569 > Security screen

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/shared/notifiers/phone_number_notifier/phone_number_state.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+@freezed
+class PhoneNumberState with _$PhoneNumberState {
+  const factory PhoneNumberState({
+    @Default(false) bool valid,
+    String? phoneNumber,
+    @Default('') String? countryName,
+    @Default('') String? countryCode,
+    @Default('') String? numCode,
+    @Default('') String? asset,
```
Comment (eli1stark) [inline lib/shared/notifiers/phone_number_notifier/phone_number_state.dart:14]: redundant nullable, remove all occurrences

File: plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart [hunk 2]
```diff
@@ -27,6 +27,9 @@ class SColorsLight implements SimpleColors {
   @override
   Color grey5 = const Color(0xFFF1F4F8);
 
+  @override
+  Color grey6 = const Color(0xFFE0EBFA);
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart:31]: investigate

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart:31]: remove colors6

File: lib/shared/notifiers/enter_phone_notifier/enter_phone_state.dart [hunk 3]
```diff
@@ -7,5 +7,6 @@ class EnterPhoneState with _$EnterPhoneState {
   const factory EnterPhoneState({
     @Default(false) bool valid,
     @Default('') String? phoneNumber,
+    @Default('+380') String? isoCode,
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/shared/notifiers/enter_phone_notifier/enter_phone_state.dart:10]: done

File: lib/shared/notifiers/phone_number_notifier/phone_number_notifier.dart [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+    state = state.copyWith(filteredCountriesCode: filteredList);
+  }
+
+  void sortClearCountriesCode() {
+    state = state.copyWith(filteredCountriesCode: []);
+  }
+
+  bool setActiveCode() {
+    if (state.phoneNumber != '' &&
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/shared/notifiers/phone_number_notifier/phone_number_notifier.dart:59]: done

File: lib/shared/notifiers/phone_number_notifier/phone_number_state.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/shared/notifiers/phone_number_notifier/phone_number_state.dart:14]: done

File: plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/colors/view/simple_colors_light.dart:31]: remove grey6

**Pull request merged**